App.js

```diff
@@ -1,26 +1,45 @@
-import React from 'react';
-import styled from 'styled-components/native';
-
-const Container = styled.View`
-	flex: 1;
-	background-color: papayawhip;
-	justify-content: center;
-	align-items: center;
-`;
-
-const Title = styled.Text`
-	font-size: 24px;
-	color: palevioletred;
-`;
+import React from 'react'
+import { StyleSheet, View, Text, ScrollView } from 'react-native'
+import { StatusBar } from 'expo-status-bar'
+import ApiFetch from './components/ApiFetch'
 
 const App = () => {
-	return (
-		<Container>
-			<Title>This is your cool app!</Title>
-			<Title>Go to App.js and start coding</Title>
-			<Title>💅💅💅</Title>
-		</Container>
-	);
-};
+  return (
+    <View style={styles.container}>
+      <ScrollView style={styles.scrollView}>
+        <Text style={styles.title}>A clever quote on your coding journey</Text>
+        <View style={styles.quoteContainer}>
+          <ApiFetch />
+        </View>
+      </ScrollView>
+    </View>
+  )
+}
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    width: '100%',
+    backgroundColor: 'papayawhip',
+    alignItems: 'center',
+    justifyContent: 'space-around',
+    paddingTop: 100
+  },
+  title: {
+    fontSize: 16,
+    color: 'palevioletred',
+    alignItems: 'center',
+    marginLeft: 30,
+    fontFamily: 'Optima-Bold'
+  },
+  quoteContainer: {
+    marginTop: 20,
+    alignItems: 'center',
+    padding: 40
+  },
+  scrollView: {
```

Comment:
How does the scrollView work here? I used it myself and had it on the container where the fetched text was so when it was very long I could scroll it. I´m not sure if I used it correctly though, so my comment here is more out of curiousity.